README.md

```diff
@@ -1,2 +1,8 @@
-# opentelemetry service
-OpenTelemetry service
+# OpenTelemetry Service
```

Comment:
great doc. Do you mind I will take pieces of it for the spec repo?

Reply:
I don't mind. Go ahead :-)